src/api/rdfData.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { getConfig } from '../config';
+import { DataFormat } from '../types/enums';
+
+const acceptHeader = header => ({
+  headers: {
+    accept: header
+  }
+});
+
+export const getRdfData = id => {
+  const url = `${
+    getConfig().informationmodelHarvester
+  }/informationmodels/${id}`;
+
+  const turtle = axios.get(url, acceptHeader(DataFormat.TURTLE));
+  const jsonld = axios.get(url, acceptHeader(DataFormat.JSONLD));
+  const rdfxml = axios.get(url, acceptHeader(DataFormat.RDF_XML));
```

Comment:
That is one option. Another option is to only make one request and change data to other formats.